src/components/CommentCard.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react'
+import { deleteComment, formatDate } from '../../utils/utils'
+
+export const CommentCard = ({ comment, user }) => {
+
+    const [deleteStatus, setDeleteStatus] = useState(`Delete comment ${comment.comment_id}?`)
+    const [deleted, setDeleted] = useState(false)
+
+    const handleDelete = (event, id) => {
+        event.target.disabled = "true"
+        setDeleteStatus(`deleting comment ${id}!`)
+        deleteComment(id)
+        .then(() => setDeleted(true))
+        .catch(err => {
+            console.log(err)
+            event.target.disabled = "false"
+            setDeleteStatus("try again...")
+        })
```

Comment:
Double check your logic for handling a failed delete - When I tried to test this, I just got a 'comment has been deleted' error